.evergreen/functions.yml

```diff
@@ -278,9 +278,8 @@ functions:
 
           echo "Running tests in Node.js runtime"
 
-          # compass-e2e-tests are ignored as we are running this test suite
-          # separately in the test-packaged-app task
-          export IGNORE_TESTS="compass-e2e-tests"
+          # Some packages are ignored as we are running these separately
+          export IGNORE_TESTS="(compass-e2e-tests|compass-smoke-tests)"
```

Comment:
As long as there isn't a test-ci in the package.json we don't need this, but I suppose it is harmless.

Reply:
Good catch. I'll revert this to avoid potential confusion and keep things simple 👍